v2/check.proto

```diff
@@ -168,9 +168,13 @@ message CheckConfig {
   // Pipelines are the pipelines this check will use to process its events.
   repeated ResourceReference pipelines = 32 [ (gogoproto.jsontag) = "pipelines" ];
 
-  repeated MetricThreshold output_metric_thresholds = 33 [ (gogoproto.jsontag) = "output_metric_thresholds,omitempty", (gogoproto.moretags) = "yaml: \"output_metric_thresholds,omitempty\"" ];
```

Comment:
I believe we need to avoid changing field numbers since they're used for serialization/deserialization. We might end up with a scenario where the data was stored with field number `33` in etcd and read back with field number 34, effectively breaking serialization.